packages/ast-spec/src/declaration/ExportAllDeclaration/fixtures/_error_/non-string-source/snapshots/1-TSESTree-Error.shot

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`AST Fixtures declaration ExportAllDeclaration _error_ non-string-source TSESTree - Error 1`] = `[TSError: Module specifier must be a string literal.]`;
+exports[`AST Fixtures declaration ExportAllDeclaration _error_ non-string-source TSESTree - Error 1`] = `"NO ERROR"`;
```

Comment:
we probably want to update the ast fixture tester so that it passes the new flag here:
https://github.com/typescript-eslint/typescript-eslint/blob/725e7f509299c28b23ec40b3a52d7c493fe64e12/packages/ast-spec/tests/util/parsers/typescript-estree.ts#L10-L16